lib/security/secureboot.pm

```diff
@@ -20,10 +20,12 @@ use version_utils qw(is_sle);
 our @EXPORT = qw(handle_secureboot);
 
 sub handle_secureboot {
-    my ($self, $sb_opt) = @_;
+    my ($self, $sb_state, $sb_opt) = @_;
```

Comment:
Here you're introducing this `sb_state` variable but where are we using it?

Reply:
Indeed that was the previous state, but that variable was even not used before 🙂

Reply:
I have no idea, this is just a revert

Reply:
Yep, looks like it